docs/developing.md

```diff
@@ -0,0 +1,58 @@
+# Frontend Shared Package
+
+The `frontend-shared` package is a library of shared utilities, components and styles common to
+multiple Hypothesis front-end applications.
+
+## Client setup
+Follow these steps to set up ``frontend-shared`` in the client repository.
+
+#### 1.Install yalc
+ Install the `yalc` tool globally if you have not already done so.
```

Comment:
```suggestion
#### 1. Install yalc

Install the `yalc` tool globally if you have not already done so.
```

Missing space after "1." and extra space before "Install". In case it is helpful at all you can use Prettier to do a certain amount of markdown formatting, although its not as comprehensive as it is for code.